src/app/tests/suites/certification/Test_TC_S_2_2.yaml

```diff
@@ -463,7 +463,7 @@ tests:
                                     [
                                         {
                                             AttributeID: 0x0000,
-                                            AttributeValue: 0x01,
+                                            ValueUnsigned8: 0x01,
```

Comment:
We need tests that send the wrong fields for the attribute ID.  Followup is best for this, since it might need test plan updates, but we absolutely must have them.

Reply:
Added a comment in https://github.com/CHIP-Specifications/chip-test-plans/issues/4060, this is the issue for adding more test cases in the Scenes test plans.